packages/types/Calendar.d.ts

```diff
@@ -97,6 +97,7 @@ export interface CalendarEvent {
   organizer: Person;
   attendees: Person[];
   additionalNotes?: string | null;
+  customInputs?: Prisma.JsonObject | null;
```

Comment:
The problem with this is that I still don't know what it is, this should be an explicit type (even if we have to define it ourselves)

Reply:
Yeah, until prisma allow us to actually use a schema for Json I think is good enough for now.